### PR TITLE
KAA-1510: Fix instruction for Ubuntu 16

### PR DESCRIPTION
**Pull request by Dubland**
/cc @vadimol

### CODE REVIEW

**Review by vadimol — APPROVED** [review]
LGTM

**Comment by vadimol** [comment]
@Dubland  please, open PR on master as well

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ wget https://sourceforge.net/projects/boost/files/boost/1.55.0/boost_1_55_0.tar.
 tar -xf boost_1_55_0.tar.gz
 cd boost_1_55_0
 ./bootstrap.sh
-sudo ./bjam install
+sudo ./bjam cxxflags=-std=c++11 install
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md:57]: Just wondering why this is needed?

Comment (ohamov) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md:57]: The option is needed to enable 11th standard support for Ubuntu 16.04 default g++ compiler. It is not enabled by default and _boost_ configuration system for some reasons does not pass the necessary flag to a compiler.

Comment (rasendubi) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md:57]: Why it was not necessary in Ubuntu 14.04?

**Pull request merged**